spec/vast_trackers.spec.js

```diff
@@ -0,0 +1,127 @@
+import { VASTClient } from '../src/vast_client';
+import { VASTTracker } from '../src/vast_tracker';
+import { inlineTrackersParsed } from '../spec/samples/inline_trackers';
+
+const vastClient = new VASTClient();
+
+describe('VASTTracker', function() {
+  let vastTracker = null;
+
+  describe('#linear', () => {
+    let spyEmitter;
+    let spyTrackUrl;
+    beforeEach(() => {
+      const ad = inlineTrackersParsed.ads[0];
+      vastTracker = new VASTTracker(vastClient, ad, ad.creatives[0]);
+      spyEmitter = jest.spyOn(vastTracker, 'emit');
+      spyTrackUrl = jest.spyOn(vastTracker, 'trackURLs');
+    });
+
+    describe('#minimize', () => {
+      beforeEach(() => {
+        vastTracker.minimize();
+      });
+
+      it('should have emitted minimize event and called trackUrl', () => {
+        expect(spyEmitter).toHaveBeenCalledWith('minimize', {
+          trackingURLTemplates: ['http://example.com/linear-minimize']
```

Comment:
Shouldn't this use the value from the ad object instead of hard coding it ? Same for all other hard coded values

Reply:
True, i replaced it.

Reply:
Actually I disagree with this change, because if one day we change the sample file and accidentally remove `trackingURLTemplates` in the unit test it would pass because `undefined === undefined`. So we could either keep the hardcoded values, or we add another `expect` to make sure the value is not undefined or null. Let's discuss this :)

Reply:
IMO the second option is better (add another expect). For instance if a macro needs to be added to any URL from the sample file, each hardcoded value will have to be updated. It is not a big deal but if we can avoid that 😅